packages/cli/src/models/network/NetworkController.ts

```diff
@@ -318,7 +318,7 @@ export default class NetworkController {
 
       Loggy.succeed(
         `upload-contract${contract.schema.contractName}`,
-        `Logic contract ${contract.schema.contractName} successfully pushed`,
+        `Logic contract ${contract.schema.contractName} successfully deployed`,
```

Comment:
Remove this mention to logic contract if we are removing it from the previous one as well